File: 프로그래머스/unrated/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.swift
import Foundation

func solution(_ arr:[Int], _ k:Int) -> [Int] {
    var ans: [Int] = []
    
    for i in 0 ..< arr.count {
        if ans.contains(arr[i]) == false && ans.count < k {
            ans.append(arr[i])
        }
    }
    
    if ans.count < k {
        for i in ans.count ..< k {
            ans.append(-1)
        }
    }
    
    return ans
}

File: 프로그래머스/0/181858. 무작위로 K개의 수 뽑기/무작위로 K개의 수 뽑기.py
def solution(arr, k):
    answer = []
    for i in arr:
        if i in answer:
            continue
        else:
            answer.append(i)
            
    
    answer = answer[:k]
    
    while len(answer) < k:
        answer.append(-1)
    
    return answer